tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -602,7 +603,7 @@ cc_library(
     hdrs = [
         "dot_op_emitter.h",
     ],
-    copts = tf_copts(),
+    copts = if_enable_acl(["-DXLA_CPU_USE_ACL=1", "-fexceptions"]),
```

Comment:
@snadampal Same here. `XLA_CPU_USE_ACL` doesn't appear in any of the source files. I don't think we need either copts here?

Reply:
> and samilarly, dot_op_emitter includes cpu_runtime that has the definitions required for acl.

@snadampal ACL-related changes in cpu_runtime.h aren't guarded with `XLA_CPU_USE_ACL`.
https://github.com/tensorflow/tensorflow/blob/6404ea36fda5b0c45ff78c440fc15cf6fd9c6ee8/tensorflow/compiler/xla/service/cpu/cpu_runtime.h#L54-L58

I think we can drop both `-DXLA_CPU_USE_ACL=1` and `-fexceptions` here.